backend/src/v1/prisma/views/pay_transparency/reports_view.sql

```diff
@@ -0,0 +1,101 @@
+SELECT
```

Comment:
What are these two .sql files used for?

Reply:
These are auto generated by the db pull @sukanya-rath